lib/spack/spack/test/cmd/create.py

```diff
@@ -0,0 +1,69 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+import argparse
+import os
+import pytest
+
+import spack.cmd.create
+import spack.util.editor
+
+from spack.main import SpackCommand
+
+
+create = SpackCommand('create')
+
+
+@pytest.fixture("module")
+def cmd_create_repo(tmpdir_factory):
+    repo_namespace = 'cmd_create_repo'
+    repodir = tmpdir_factory.mktemp(repo_namespace)
+    repodir.ensure(spack.repo.packages_dir_name, dir=True)
+    yaml = repodir.join('repo.yaml')
+    yaml.write("""
+repo:
+    namespace: cmd_create_repo
+""")
+
+    db = spack.repo.RepoPath(str(repodir))
+    with spack.repo.swap(db):
+        yield repodir
+
+    # shutil.rmtree(str(repodir))
+
+
+@pytest.fixture(scope='module')
+def parser():
+    """Returns the parser for the module"""
+    prs = argparse.ArgumentParser()
+    spack.cmd.create.setup_parser(prs)
+    return prs
+
+
+@pytest.fixture
+def mock_editor(monkeypatch):
```

Comment:
Is this fixture being used?

Reply:
Not anymore.  Thanks for catching this one too.